### DISCUSSION TITLE
How to find the tech events ?

### DISCUSSION THREAD

**Discussion by AthulKkumar** [community/community#53899] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

In github education they are listing some events but nowdays they are not listing any new events. So how do i find about new tech events .

**Comment by multikitty** [comment]
Hi, @AthulKkumar 
Github is primarily a platform for software development and collaboration. While Github Education might list some tech events, it is not the main source for finding events.
Keep in mind that Github might not be the best platform for finding tech events, so it's always a good idea to look at other sources as well.

**Comment by ncerutti** [comment]
Hi! I would suggest having a look at meetup.com. It is mainly focused on live events, but there is plenty of online-only events too. I am not affiliated in any way, but I joined a couple of very nice communities thanks to that :)
